packages/vscode-extension/lib/expo_router_plugin.js

```diff
@@ -14,7 +14,7 @@ function useRouterPluginMainHook({ onNavigationChange, onRouteListChange }) {
   const pathname = routeInfo?.pathname;
   const params = routeInfo?.params;
 
-  const filteredParams = params ?? {};
+  const filteredParams = getParamsWithoutDynamicSegments(routeInfo);
```

Comment:
would it make sense to also handle deleting of the __EXPO_ROUTER_key property in this newly added method given we call it from all the versions of the router plugin?